identity/src/ecdsa.rs

```diff
@@ -102,17 +109,28 @@ impl SecretKey {
         signature.as_bytes().to_owned()
     }
 
-    /// Encode a secret key into a byte buffer.
+    /// Convert a secret key into a byte buffer.
     pub fn to_bytes(&self) -> Vec<u8> {
         self.0.to_bytes().to_vec()
     }
 
     /// Decode a secret key from a byte buffer.
-    pub fn from_bytes(buf: &[u8]) -> Result<Self, DecodingError> {
+    #[deprecated(
+        since = "0.2.0",
```

Comment:
What about `since` tags in `Keypair`? That's the reason why I did this.

Reply:
The webpage constantly move the focus to this conversation and I really don't know why. I canceled it and it pop up again.

Reply:
It's so annoying, I'll mark this as resolved to see what happens.

Reply:
> What about `since` tags in `Keypair`? That's the reason why I did this.

Yes, I added this at a time when I didn't fully understand what it does. I've since learned that we don't strictly need them and it easy to get them wrong with releases happening between starting and merging a PR. Hence, I am now tending to not include them because they are only for docs purposes we mostly just want users to show them transition paths via the compiler.